airbyte-webapp/src/hooks/services/Feature/FeatureService.test.tsx

```diff
@@ -162,6 +162,26 @@ describe("Feature Service", () => {
       rerender({ overwrite: undefined });
       expect(result.current.sort()).toEqual([FeatureItem.AllowDBTCloudIntegration, FeatureItem.AllowSync]);
     });
+
+    describe("env variable overwrites", () => {
+      beforeEach(() => {
+        process.env.REACT_APP_FEATURE_ALLOW_SYNC = "false";
+        process.env.REACT_APP_FEATURE_ALLOW_CHANGE_DATA_GEOGRAPHIES = "true";
+      });
+
+      afterEach(() => {
+        (process.env.NODE_ENV as string) = "test";
```

Comment:
:information_source: Need to cast this, since this is by default `readonly` so we can't overwrite it.